### ISSUE TITLE
Add Last-Modified header

### ISSUE THREAD

**Issue by jfinkels** [jfinkels/flask-restless#190]
and possible ETag?

**Comment by jfinkels** [comment]
I think this is out-of-scope for Flask-Restless, as it depends on how the user implements the last update time in the model. It can be implemented by a preprocessor, which could check the headers, the last-update time, the etag, etc., then raise a `ProcessingException` with the appropriate status code (e.g. `304 Not Modified`). This may be valuable as an example or in the documentation, so I'm marking this as a documentation issue.

**Comment by TrilceAC** [comment]
Agree. I like what @jfinkels proposes because you don't have to modify your model or add a new field just because you want this common feature. Instead, considering that you have some timestamp field, you only have to follow a recipe.

Another way to do this, could be by somehow implementing a function, maybe a decorator, that given a model field, it generates the header info and the proper checker to check whether it has been modified or not. I like this approach too, because you offer the functionality, but you are free to choose which field (or even fields) are those that need to be checked before updating.
